flexmeasures/ui/templates/assets/asset_properties.html

```diff
@@ -245,152 +258,1077 @@ <h3>All child assets for {{ asset.name }}
+
+                    card.appendChild(cardHighlight);
+                    scrollContainer.appendChild(card);
+                };
+
+                return scrollContainer;
+            }
+
+            async function renderFlexModelFieldCard(name, value) {
+                const card = document.createElement('div');
+                card.className = 'card m-1 mt-0';
+                card.id = `${name}-control`;
+
+                card.onclick = function () {
+                    if (card.classList.contains('border-on-click')) {
+                        if (activeCard() && activeCard().id === card.id) {
+                            if (selectedIndex() === null && !FlexModelFieldValidTypes[name].includes(Array)) {
+                                setActiveCard(null); // Deselect if the same card is clicked again
+                                card.classList.remove('border-on-click'); // Remove the border class
+                            }
+                        }
+                        return;
+                    }
+
+                    const allCards = document.querySelectorAll('.card');
+                    allCards.forEach(c => c.classList.remove('border-on-click'));
+                    card.classList.add('border-on-click');
+                    setActiveCard(card); // Set the active card
+                    renderSelectInfoCards(name); // Render the info card for the selected field
+                    if (!FlexModelFieldValidTypes[name].includes(Array)) {
+                        renderFlexInputOptions(); // Render the input options for the selected field
+                    }
+                }
+
+                const isString = typeof value === "string";
+                const isArray = Array.isArray(value);
+                const isBool = typeof value === "boolean";
+                const isNumber = typeof value === "number";
+                const isKeyValueObject = typeof value === "object" &&
+                    value !== null &&
+                    !Array.isArray(value) && // Exclude arrays
+                    Object.keys(value).length > 0;
+
+                const highlightCard = document.createElement('div');
+                highlightCard.className = 'card-highlight';
+                card.appendChild(highlightCard);
+                const titleContainer = document.createElement('div');
+                titleContainer.className = 'd-flex justify-content-between align-items-center';
+                const title = document.createElement('h5');
+                title.className = 'card-title';
+                title.textContent = getFlexFieldTitle(name);
+                const titleSubContainer = document.createElement('div');
+                titleSubContainer.className = 'd-flex'
+                titleSubContainer.appendChild(title);
+
+
+                if (isArray) {
+                    const plusIcon = document.createElement('i');
+                    plusIcon.className = 'fa fa-solid fa-plus ps-1';
```

Comment:
Maybe add a `title` attribute that says what will happen, like "Add another setting"

Reply:
Agreed, we should be consistent.

Users probably understand setting better than item. Item sounds more like a physical thing.